lib/widgets/auth/email_auth_options_page/links/sign_in_link.dart

```diff
@@ -14,13 +13,8 @@ class SignInLink extends StatelessWidget {
                 decoration: TextDecoration.underline,
                 color: Colors.blue)),
         onTap: () {
-          Form.of(context).reset();
           context.dispatch(UpdateEmailAuthOptionsPage(
             mode: EmailAuthMode.signIn,
-            email: '',
-            password: '',
-            repeatPassword: '',
-            autovalidate: AutoValidate.disabled,
```

Comment:
ran into a problem while manual testing, 
if the user navigates between sign in/sign up screens **without** entering any text, autovalidation is still set to Autovalidate.onUserInput. I know we want those UX points so I suggest we only set autovalidation (after navigation between screens) if the user has entered text

Reply:
sorry if this wasnt the clearest, im blaming 4pm on a friday 😅. happy to discuss if wsnt clear

Reply:
super clear, thanks mate!  I'll have a crack at fixing

Reply:
I wasn't able to repro on iPhone emulator or a Pixel4, so I'm going to push this through and add an Asana task for Tam to check (some time after Monday) if the problem still exists.  Thanks!